db/seeds.rb

```diff
@@ -11,57 +11,93 @@
     u.email = "dev@hfeyp.com"
     u.password = "Partyw0rd"
   end
+  User.create! do |u|
+    u.email = "guy@home.com"
+    u.password = "guyguy"
+  end
 end
 
+# rubocop:disable Layout/HeredocIndentation
```

Comment:
Probably better to tell rubocop to exclude this file given it's one big exclusion.

Reply:
Lets keep Rubocop for the other seed content, its just HERE documents is conflicts with